rebasehelper/helpers/bugzilla_helper.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+#
+# This tool helps you rebase your package to the latest version
+# Copyright (C) 2013-2019 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Authors: Petr Hráček <phracek@redhat.com>
+#          Tomáš Hozza <thozza@redhat.com>
+#          Nikola Forró <nforro@redhat.com>
+#          František Nečas <fifinecas@seznam.cz>
+
+import logging
+import os
+import re
+import shutil
+from typing import cast, Tuple
+
+import git  # type: ignore
+import requests
+
+from rebasehelper.exceptions import RebaseHelperError
+from rebasehelper.helpers.input_helper import InputHelper
+from rebasehelper.logger import CustomLogger
+
+
+logger: CustomLogger = cast(CustomLogger, logging.getLogger(__name__))
+
+
+class BugzillaHelper:
+
+    """Class for working with bugs by Upstream Release Monitoring on bugzilla."""
+
+    @classmethod
+    def prepare_rebase_repository(cls, bugzilla_id: str) -> Tuple[str, str]:
+        """Clones a repository based on Upstream Release Monitoring bug.
+
+        Args:
+            bugzilla_id: ID of the bug.
+
+        Returns:
+            Name of the cloned package and version to rebase to.
+
+        """
+        bug_data = requests.get('https://bugzilla.redhat.com/rest/bug/{}'.format(bugzilla_id)).json()
+        if 'error' in bug_data:
+            raise RebaseHelperError("No bug with ID {} was found".format(bugzilla_id))
+        bug = bug_data['bugs'][0]
+        if bug['creator'] != 'Upstream Release Monitoring':
+            raise RebaseHelperError('The given bug was not created by Upstream Release Monitoring')
+        component = bug['component'][0]
+        match = re.match(r'^.*-(?P<version>.*) is available$', bug['summary'])
+        if match:
+            version = match.group('version')
+        else:
+            raise RebaseHelperError('Failed to parse new version from the given bug')
+
+        url = 'https://src.fedoraproject.org/rpms/{}.git'.format(component)
```

Comment:
This shouldn't be hardcoded. It could change (it already has at least once in the recent history), a constant would be better.